### ISSUE TITLE
Pulling from a Google Form

### ISSUE THREAD

**Issue by castellanosr** [Edurate/edurate#3]
We need to create a Google form that is eventually able to be customized for a single class. Then this form needs to be downloaded, turned into a spreadsheet and run through the edurate program

**Comment by dillam** [comment]
I want this issue

**Comment by ZachAndrews98** [comment]
Does this include reading the file of responses or is that another issue we need to make?

**Comment by dillam** [comment]
This won't being reading from a file so much as taking the responses straight from Google sheet so the google sheet won't need to be downloaded. That is if we want to do this the same way as my group did in GatorGrouper.

**Comment by dillam** [comment]
I made a branch for this issue. The code that is in there takes data from the google sheet and puts it into a csv. Right now, it doesn't output the data in the same order as the questions. Maybe someone else can look into that. It also needs to take arguments from the command line so once command line verification is done, we need to allow the function to take arguments for output file name and google sheet name.

**Comment by toccinAC** [comment]
Do you know what the output will be so the NLP group will be able to better anticipate what we will be manipulating?

**Comment by burrowss** [comment]
Do we want to look into creating a new google form everytime a response is submitted? As of now, the spreadsheet will have multiple entries from the same person based on our testing which will throw off analysis

**Comment by millinj2** [comment]
There is the option in the settings of the google form that you can only allow one reponse. Do you think this would work?

**Comment by burrowss** [comment]
@ZachAndrews98 and I figured it out last night. We have it so that it only gets the most recent response based on date in the program itself
